### ISSUE TITLE
Undefined StyleSheet not showing as error

### ISSUE THREAD

**Issue by xahon** [Intellicode/eslint-plugin-react-native#169]
If you use `StyleSheet.create({})` without `StyleSheet` import it causes no error

**Comment by Intellicode** [comment]
Thanks for reporting this request. Currently, this is not covered by any existing lint rules by the plugin. A new one would be an option. If it is something you encounter often, can you make a PR that covers this case?

**Comment by xahon** [comment]
No, i just randomly found that. If i will have enough time i'll try

**Comment by agarcia17** [comment]
@xahon, could it be that you have "browser": true in your eslint env property? Cause I believe browsers have a global "StyleSheet" variable

**Comment by xahon** [comment]
@agarcia17 yes, browser set to true

**Comment by agarcia17** [comment]
@xahon, try setting it to false and restart VS Code and see if the issue still persist

**Comment by xahon** [comment]
@agarcia17 I'm not using vs code, but vim. I'll check it later
